placideirandora/epic-mail
# #167095253 refactored controllers to implement async/await instead of .then promises

#### What does this PR do?
It implements async/await instead of .then promises.

#### Description of Task to be done?
1. It refactors the controllers to implement async/await promise approach instead of the .then one.
2. It makes some necessary changes like updating the http response code and right response message.
3. It updates the tests to reflect on the changes made

#### How should this be manually tested?
1. Clone the repo.
2. Follow the instructions on how to set up the repo in the ReadMe.
3. Check to see if the controllers are now implemented with async/await.
4. You can also check if the tests pass successfully.
